### PR TITLE
Watch absolute paths

### PR DESCRIPTION
**Pull request by dougkoch**
Fixes: #49

### CODE REVIEW

File: ibazel/ibazel.go [hunk 1]
```diff
@@ -99,6 +106,35 @@ func New() (*IBazel, error) {
...
+			path = volume
+		}
+
+		// Check if we're at the workspace path
+		if _, err := os.Stat(filepath.Join(path, "WORKSPACE")); !os.IsNotExist(err) {
+			return path, nil
+		}
+
+		// If we've reached the root, then we know the cwd isn't within a workspace
+		if path == volume {
```
Comment (achew22) [inline ibazel/ibazel.go]: This should fail when called from inside a test. If it doesn't then it is depending on a fragile implementation detail of Bazel's test launcher that will change in the future and break.

**Comment by achew22** [comment]
#89 should also function as a template for writing an e2e test. Take the simple test, cd into a subdir, start ibazel, cd back and then run the rest of the test.

File: ibazel/ibazel.go (hunk 1 above)
Comment (dougkoch) [reply ibazel/ibazel.go]: Are you implying that this should just default to the cwd if a WORKSPACE isn't found (and let bazel give the error)? Or would you suggest some other way to keep this from failing in a test?

Comment (achew22) [reply ibazel/ibazel.go]: I think this is an interface, "WORKSPACEFinder", or something and that there can be a trivial testing fake that returns $CWD or empty string or something as the path.

**Pull request merged**